compiler/src/dotty/tools/repl/ReplDriver.scala

```diff
@@ -235,6 +235,9 @@ class ReplDriver(settings: Array[String],
             val newImports = newState.imports ++ extractImports(parsed.trees)
             val newStateWithImports = newState.copy(imports = newImports)
 
+            implicit val ctx: Context = newState.run.runContext
```

Comment:
I don't think this implicit context is needed.

Reply:
You are right, an implicit State is required.